HACKING.rst

```diff
@@ -38,33 +38,34 @@ you may need to run::
   pip install importlib
 
 
-**Testing the HaaS**
-====================
+`Testing.md <docs/testing.md>`_ contains more information about testing HaaS.
 
+Configuring HaaS
+================
 
 Now the ``haas`` executable should be in your path.  First, create a
 configuration file ``haas.cfg``. There are two examples for you to work from,
-``haas.cfg.dev-example``, which is oriented towards developemnt, and
-``haas.cfg.example`` which is more production oriented. These files are well
-commented; read them carefully. In relation to testing, one option is worth
-calling out specifically: the ``dry_run`` option in the ``[devel]`` section,
-which allows most unit tests (which only touch the database) to be run on one's
-local machine, without needing to have a full HaaS setup available.
-
-Next initialize the database with the required tables, with ``haas
-init_db``.  Run the server with ``haas serve``.  Finally, see ``haas help``
-for the various API commands one can test.  Here is an example session,
-testing ``headnode_delete_hnic``::
+``examples/haas.cfg.dev-example``, which is oriented towards development, and
+``examples/haas.cfg.example`` which is more production oriented.
+
+HaaS can simulate nodes, headnodes and networks without needing physical
+hardware to run them on. For simulating nodes and headnodes, set ``dry_run =
```

Comment:
This is *very* misleading -- it doesn't "simulate" anything -- it just noops them. It's important that the developer understands this.